code/modules/admin/verbs/SDQL2/SDQL_2_wrappers.dm

```diff
@@ -214,6 +214,9 @@
 /proc/_step_away(ref, trg, max)
 	step_away(ref, trg, max)
 
+/proc/_file(name)
+	return file(name)
```

Comment:
This seems like a massive security hole to allow admins to call, what is this doing here